solr/core/src/java/org/apache/solr/cli/DeleteTool.java

```diff
@@ -198,13 +187,13 @@ protected void deleteCollection(CloudSolrClient cloudSolrClient, CommandLine cli
 
     String configName =
         zkStateReader.getClusterState().getCollection(collectionName).getConfigName();
-    boolean deleteConfig = true;
+    boolean deleteConfig = false;
     if (cli.hasOption("delete-config")) {
-      deleteConfig = "true".equals(cli.getOptionValue("delete-config"));
+      deleteConfig = true;
     } else if (cli.hasOption("d")) {
-      deleteConfig = "true".equals(cli.getOptionValue("d"));
+      deleteConfig = true;
     } else if (cli.hasOption("deleteConfig")) {
-      deleteConfig = "true".equals(cli.getOptionValue("deleteConfig"));
+      deleteConfig = true;
```

Comment:
Is it more readable if replaced by the following?
```suggestion
    boolean deleteConfig = cli.hasOption("delete-config")
        || cli.hasOption("deleteConfig")
        || cli.hasOption("d");
```

Reply:
Love it...  I can't merge directly from the UI for some reason so applying manually.